Loop/Managers/DeviceDataManager.swift

```diff
@@ -364,6 +370,38 @@ final class DeviceDataManager: CarbStoreDelegate, CarbStoreSyncDelegate, DoseSto
         }
     }
 
+    /// NonMySentry Battery Calculation for Alkaline and Lithuim  #141
+    ///
+    /// - parameter currVoltage: Current Voltage Reading from Pump
+    ///
+    public var x22BatteryPercentRemaining : Double = -1
```

Comment:
As I understand it you are saying I should write the Battery % Remaining to dataManager.latestPumpStatusFromMySentry.batteryRemainingPercent.  I totally agree reuse of the already existing structure would be great, however I am unsure how to write to the batteryRemainingPercent var.

`latestPumpStatusFromMySentry?.batteryRemainingPercent = ((currVoltage - minVoltage)/(maxVoltage - minVoltage))`

I receive an error

`Cannot assign to property: 'batteryRemainingPercent' is a 'let' constant`